themes/bootstrap3/templates/myresearch/mylist.phtml

```diff
@@ -101,7 +101,7 @@
   <?php endif; ?>
 </div>
 
-<div class="<?=$this->layoutClass('sidebar')?>" id="myresearch-sidebar">
+<div class="<?=$this->layoutClass('sidebar')?>" id="myresearch-sidebar" role="navigation" aria-label="<?=$this->transEsc('Your Account')?>">
```

Comment:
If you're going to do this here, I would think that you would also want to make matching changes to all of the templates that make a renderInContext call to import the `myresearch/menu.phtml` template. (See, for example, the schedulesearch.phtml template below where we talked about the unexpected/unexplained whitespace changes... and there are quite a few others too).

Reply:
For this I have too deeply apologize, I did those changes in a hurry, you are right. I shall set the whitespace/formatting right for all calls and add the proper changes